packages/core/useRafFn/index.ts

```diff
@@ -58,6 +58,7 @@ export function useRafFn(fn: (args: UseRafFnCallbackArguments) => void, options:
     const delta = timestamp - (previousFrameTimestamp || timestamp)
 
     if (intervalLimit && delta < intervalLimit) {
+      previousFrameTimestamp = timestamp
```

Comment:
I think we should move `previousFrameTimestamp = timestamp` to up line, and remove line 68.

Reply:
True, that makes a lot more sense! I've updated my PR, thanks!